README.md

```diff
@@ -79,3 +79,18 @@ jobs:
     test: false
     mathlib-cache: false
 ```
+
+## Keep the action updated with `dependabot`
+
+In order to receive notification when this action is updated (and any others used in your repo), add the file `.github/dependabot.yml` with the contents:
```

Comment:
I think it makes sense to give more explanation of why this is recommended and how it works. Something like,

```suggestion
Because Lean is under heavy development, changes to Lean or Lake could break outdated versions of `lean-action`. You can configure [dependabot](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates) to automatically create a PR to update `lean-action` when a new stable version is released. 

Here is an example `.github/dependabot.yml` which configures `dependabot` to check daily for updates to all GitHub actions in your repository:
```

Reply:
Very wise to add the motivation. It's a bit of a nightmare how unstable the user interface of lake and lean is. At least this mostly extracts the problem into updating the action.